### PR TITLE
Stephen's nits

### PR DESCRIPTION
**Pull request by gselander**
#377

### CODE REVIEW

**Comment by sftcd** [comment]
Reading through those changes now will note things arising here, first one is below.

If an EAD item can occur more than once, do all occurrences of a given ead_label in one message or one session need to have the same criticality? Presumably if any are marked critical then all could be, and vice-versa, so not sure if mixing is needed, and it might lead to unexpected outcomes maybe?

**Comment by sftcd** [comment]
Actually that's my only comment on these changes - otherwise they look good.

**Comment by gselander** [comment]
@sftcd Please check fc60780

File: draft-ietf-lake-edhoc.md [hunk 1]
```diff
@@ -315,15 +315,15 @@ In order to create a "full-fledged" protocol some additional protocol elements a
 
 * A keying material exporter and a key update function with forward secrecy.
 
-* Verification of the selected cipher suite.
+* Negotiation of cipher suite, more specifically verification by Responder of most preferred cipher suite by Initiator
```
Comment (emanjon) [inline draft-ietf-lake-edhoc.md:318]: "more specifically verification by Responder of most preferred cipher suite by Initiator"

Not sure this makes thing easier to understand. This sentence is not correct, but a correct sentence would be even longer and complicated....

File: draft-ietf-lake-edhoc.md [hunk 2]
```diff
@@ -529,18 +529,18 @@ ID_CRED_x may contain the authentication credential CRED_x, but for many setting
 
 EDHOC relies on COSE for identification of credentials and supports all credential types for which COSE header parameters are defined including X.509 certificates ({{I-D.ietf-cose-x509}}), C509 certificates ({{I-D.ietf-cose-cbor-encoded-cert}}), CWT ({{cwt-header-param}}) and CWT Claims Set (CCS) ({{cwt-header-param}}).
 
-ID_CRED_I and ID_CRED_R are COSE header maps and contains one or more COSE header parameters. ID_CRED_I and ID_CRED_R MAY contain different header parameters. The header parameters typically provide some information about the format of the credential.
+ID_CRED_I and ID_CRED_R are of type COSE header_map, as defined in Section 3 of {{RFC9052}}, and contains one or more COSE header parameters. ID_CRED_I and ID_CRED_R MAY contain different header parameters. The header parameters typically provide some information about the format of the credential.
```
Comment (emanjon) [inline draft-ietf-lake-edhoc.md:532]: That is good!

File: draft-ietf-lake-edhoc.md [hunk 3]
```diff
@@ -529,18 +529,18 @@ ID_CRED_x may contain the authentication credential CRED_x, but for many setting
...
 
-Note that COSE header parameters in ID_CRED_x are used to identify the message sender's credential. There is therefore no reason to use the "-sender" header parameters, such as x5t-sender, defined in Section 3 of {{I-D.ietf-cose-x509}}. Instead, the corresponding parameter without "-sender", such as x5t, SHOULD be used.
-
-Example: X.509 certificates can be identified by a hash value using the 'x5t' parameter:
+Example: X.509 certificates can be identified by a hash value using the 'x5t' parameter, see Section 2 of {{I-D.ietf-cose-x509}}:
```
Comment (emanjon) [inline draft-ietf-lake-edhoc.md:534]: This seems bad. Why? EDHOC is using COSE header_map but changing the definition of some of the parameters. Why should that not be described?

**Review by emanjon — APPROVED** [review]

File: draft-ietf-lake-edhoc.md (hunk 1 above)
Comment (gselander) [reply draft-ietf-lake-edhoc.md]: Abbreviated, see 67947ff below.

**Pull request merged**